src/main/resources/db/migration/V20211001_21__HMAN-21-HearingService.sql

```diff
@@ -223,7 +223,7 @@ CREATE TABLE public.organisation_detail (
                       tech_party_id bigint not null,
                       organisation_name varchar(2000) not null,
                       organisation_type_code varchar(60) not null,
-                      hmcts_organisation_reference varchar(60) not null
+                      hmcts_organisation_reference varchar(60)
```

Comment:
For database changes we need to create a new migration script with only the change of relevance, otherwise this will break for anyone who has already run this existing script. Happy to discuss/explain the process!

Reply:
updated in 0ae612a810d04011ea9c0e28759e449f3a4fa43d